__tests__/components/editor/property/InputListLOC.test.js

```diff
@@ -108,52 +174,6 @@ describe('<Typeahead /> component', () => {
     expect(wrapper.find('#targetComponent').props().placeholder).toMatch('Frequency (RDA 2.14)')
   })
 
-  describe('default values', () => {
```

Comment:
since we are using selectorReducer function to get the defaults from the propertyTemplate, there is no need to test this here.